CRUD/service/connection.hpp

```diff
@@ -77,7 +77,7 @@ class connection
                 request_, buffer_.data(), buffer_.data() + bytes_transferred);
             auto itr = std::find_if( request_.headers.begin(), 
                                      request_.headers.end(), 
-                                     [](const header &h) { return h.name == "Content-Length" ; }
+                                     [](const header &h) { return h.name == "content-length" ; }
             ) ;
             if ( itr != request_.headers.end()) {
                 request_.data = std::string(data, boost::lexical_cast<long>(itr->value)) ;
```

Comment:
ok , I will merge your changes from CRUD into vanilla-rtb directly , I think it's cleaner that way and then if no conflicts , then I will accept your pending pull request from vanilla-rtb , so let's see how it works.
Thank you , Arseny !